src/Sepes.Infrastructure/Service/Azure/AzureApiServiceBase.cs

```diff
@@ -95,9 +108,30 @@ protected async Task<T> GetResponse<T>(string url, bool needsAuth = true, Cancel
                     if (needsAuth)
                     {
                         apiRequestClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", token);
-                    }                   
+                    }
```

Comment:
needsAuth er alltid true? Er det poeng med en IF på den?

Reply:
Somebody also sends false as an argument, like AzureCostManagementService which implements that class